react/Nav/styles.styl

```diff
@@ -12,8 +12,10 @@
 .c-nav-link
     @extend $nav-link
 
+.c-nav-icon // Deprecated
 .c-nav-item__icon
     @extend $nav-item-icon
 
+.c-nav-text // Deprecated
 .c-nav-item__text
```

Comment:
Same here

Reply:
Well, the react component doesn't use the same classes and every app doesn't use the React component, I didn't want to make it a breaking change out of it and block UI updates just for a classname change.

Reply:
> the react component doesn't use the same classes

Yes, but we can change the component so it uses the same classes as the vanilla css version. It it not a breaking change as the classnames are private to the component and not part of its API.

Reply:
Oh yes you mean the other way around… true